testing-unit-common/src/main/kotlin/com/bumble/appyx/testing/unit/common/helper/NodeTestHelper.kt

```diff
@@ -19,11 +19,6 @@ open class NodeTestHelper<N : Node>(private val node: N) {
         require(state != Lifecycle.State.INITIALIZED) {
             "Can't move to INITIALIZED state"
         }
-        require(
```

Comment:
Being able to move to DESTROYED is a valid test case, and the Node class itself has a safeguard to prevent moving from a destroyed state to another state.